cms/envs/bok_choy.py

```diff
@@ -25,6 +25,8 @@
 # and (b) that the files are the same in Jenkins as in local dev.
 os.environ['SERVICE_VARIANT'] = 'bok_choy_docker' if 'BOK_CHOY_HOSTNAME' in os.environ else 'bok_choy'
 os.environ['CONFIG_ROOT'] = path(__file__).abspath().dirname()
+os.environ['STUDIO_CFG'] = str.format("{config_root}/{service_variant}.yml",
```

Comment:
Does this syntax work? I've only ever seen this used as `" ".format(var="123")`

Reply:
Oh yeah, that was a mistake, I must not've saved